.github/bin/label_abandoned_prs.py

```diff
@@ -12,6 +12,13 @@
     " this PR will be labeled as Abandoned PR - Needs New Owner and"\
     " either another developer will finish the PR or it will be closed."
 
+today = datetime.datetime.now()
+two_weeks = timedelta(weeks=2)
+twelve_days = timedelta(days=12)
+two_days = timedelta(days=2)
+eastern = datetime.timezone(datetime.timedelta(hours=-5))
+et_today = today.astimezone(eastern)
+
 for json_data in json_output:
     already_warned = False
     string = json_data['updatedAt']
```

Comment:
rename this to prevent confusion, bad variable name

Reply:
oh yeah i didn't see that, i'll rename it to `time_string`